tqdb/parsers/equipment.py

```diff
@@ -182,7 +187,9 @@ def is_valid_classification(self, dbr, dbr_file, result):
             if classification == 'Rare':
                 file_name = os.path.basename(dbr_file).split('_')
                 if len(file_name) < 2 or file_name[1] not in DIFFICULTIES:
-                    raise StopIteration
+                    raise InvalidItemError(f"File name {file_name} does not "
+                                        f"specify difficulty, or difficulty "
```

Comment:
Don't think you need f-strings as continuations if they're not containing any variables etc.